src/runtime/linux/mod.rs

```diff
@@ -17,11 +20,18 @@ fn detect_features_impl<T: FeatureQuery>(x: T) -> usize {
     {
         super::aarch64::detect_features(x)
     }
+    #[cfg(target_arch = "powerpc64")]
+    {
+        super::powerpc64::detect_features(x)
+    }
 }
 
 /// Detects ARM features:
 pub fn detect_features() -> usize {
-    // FIXME: use libc::getauxval, and if that fails /proc/auxv
```

Comment:
We should keep this fixme because we are still not calling `libc::getauxval`. We should try to do that first, and if that fails, then read `/proc/self/auxv`, and if that fails, then try to read `/proc/cpuinfo`.